packages/dumb/src/components/settings/history/history-table.util.tsx

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** The file extension should be `.ts` and not `.tsx`‏